config/environments/production.rb

```diff
@@ -34,7 +34,7 @@
   # Mount Action Cable outside main process or domain
   # config.action_cable.mount_path = nil
   # config.action_cable.url = 'wss://example.com/cable'
-  # config.action_cable.allowed_request_origins = [ 'http://example.com', /http:\/\/example.*/ ]
+  config.action_cable.allowed_request_origins = ENV['RAILS_ALLOWED_REQUEST_ORIGINS'].split(',').map(&:strip)
```

Comment:
if it's not configured then this would block everything right ?
```ruby
if origins = ENV['RAILS_ALLOWED_REQUEST_ORIGINS'].to_s.split(',').map(&:strip).presence
  config.action_cable.allowed_request_origins = origins
end

```

Reply:
True, nice :)

Reply:
Actually I guess it would set `allowed_request_origins` to `nil`, which... might break things?